### DISCUSSION TITLE
Import and Export Data

### DISCUSSION THREAD

**Discussion by gaLDWoRE** [teslamate-org/teslamate#451] [Ideas]
**Is your feature request related to a problem? Please describe.**

Not really a problem, but before starting to use TeslaMate I'm using Stats on iOS, which allows you to export some data, like Battery Health, Distance Driven, Efficiency, Phantom drain. 

**Describe the solution you'd like**

A general import and export csv file that we can either fill in with our data manually and then import. Same then would go for exporting via csv, so that we can manipulate the data in Excel as needed.

**Describe alternatives you've considered**

Tried to find where data is stored in something like Tesla API Scraper, but wasn't able to successfully retreat the data into a csv. But where the data is stored so I can modify on TeslaMate, I don't even know.

**Comment by ghost** [comment]
Ability to import from Stats would be great. If the dev wants to reach out to me, I can forward some of the exported data so they can see what they'd be working with.

**Comment by sw1002** [comment]
Please add the ability to export drive data into TeslaFi Format
